scripts/rust-toolchain.sh

```diff
@@ -2,4 +2,4 @@
 
 # script is used by github workflows
 
-grep '^channel =' rust-toolchain.toml | sed 's/.*= "\(.*\)"/\1/'
+grep 'CARGO_TOOLCHAIN: .*nightly' laze-project.yml | grep -o 'nightly-[0-9-]\+'
```

Comment:
This looks a bit fragile, what's the best route here?

* Allow laze to source variables from a file? `env: global: CARGO_TOOLCHAIN: < our-toolchain.txt` (YAML may have something to offer, probably not file access but distinguishing an action from a string)
* Allow laze to export a variable? (`laze dump-var global CARGO_TOOLCHAIN`?)
* Just hope that this never breaks in unexpected non-CI-failing ways?